ue4cli/JsonDataManager.py

```diff
@@ -12,6 +13,19 @@ def __init__(self, jsonFile):
 		Creates a new JsonDataManager instance for the specified JSON file
 		"""
 		self.jsonFile = jsonFile
+
+	def loads(self):
+		"""
+		Reads and loads owned jsonFile
+		"""
+		try:
+			path = self.jsonFile
+			file = Utility.readFile(path)
+			return json.loads(file)
+		except json.JSONDecodeError as e:
+			# FIXME: This is the only place outside of Utility class where we use UtilityException.
+			# Not worth to create new Exception class for only one single case, at least not now.
+			raise UtilityException(f'failed to load "{str(path)}" due to: ({type(e).__name__}) {str(e)}')
```

Comment:
Just to quickly highlight here (and a few other places I noticed) when one `raise`s inside an exception handler, that should either be `raise ... from e` to chain the existing exception as a sub-exception "direct cause", or `raise ... from None` to indicate that the original exception is now not-interesting.

A plain `raise ...` inside the exception handler in Python indicates that [exception handling _failed_ and generated a new exception](https://docs.python.org/3/tutorial/errors.html#exception-chaining), which isn't what we're doing here.

Reply:
Sorry for missing this. Fixed with https://github.com/adamrehn/ue4cli/pull/65/commits/e58d718953499b063d196eb145078d10381a933c
I'll resolve this thread. If something is still missing, please create another one.

Thanks!